setup.cfg

```diff
@@ -4,7 +4,5 @@ dev = develop easy_install lasagne[testing]
 [pytest]
 addopts =
     -v --doctest-modules
-    --cov=lasagne --cov-report=term-missing
```

Comment:
@benanne Sorry it's a bit unclear but I moved coverage over to `.travis.yml` instead. The thing is pycharm is a bit grumpy and the debugger breaks when `--cov` is in there. I thought this was a good compromise.